Samples/PublicSamples/RecordingBot/docs/tutorials/deploy/aks.md

```diff
@@ -0,0 +1,719 @@
+  "enableEncryptionAtHost": false,
+  "enableFips": false,
+  "enableNodePublicIp": false,
+  "enableUltraSsd": false,
+  "gpuInstanceProfile": null,
+  "hostGroupId": null,
+  "id": "/subscriptions/yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyyy/resourcegroups/recordingbottutorial/providers/Microsoft.ContainerService/managedClusters/recordingbotcluster/agentPools/nodepool1",
+  "kubeletConfig": null,
+  "kubeletDiskType": "OS",
+  "linuxOsConfig": null,
+  "maxCount": null,
+  "maxPods": 30,
+  "minCount": null,
+  "mode": "System",
+  "name": "nodepool1",
+  "networkProfile": null,
+  "nodeImageVersion": "AKSUbuntu-2204gen2containerd-202403.25.0",
+  "nodeLabels": null,
+  "nodePublicIpPrefixId": null,
+  "nodeTaints": null,
+  "orchestratorVersion": "1.28.5",
+  "osDiskSizeGb": 128,
+  "osDiskType": "Managed",
+  "osSku": "Ubuntu",
+  "osType": "Linux",
+  "podSubnetId": null,
+  "powerState": {
+    "code": "Running"
+  },
+  "provisioningState": "Succeeded",
+  "proximityPlacementGroupId": null,
+  "resourceGroup": "recordingbottutorial",
+  "scaleDownMode": null,
+  "scaleSetEvictionPolicy": null,
+  "scaleSetPriority": null,
+  "spotMaxPrice": null,
+  "tags": null,
+  "type": "Microsoft.ContainerService/managedClusters/agentPools",
+  "typePropertiesType": "VirtualMachineScaleSets",
+  "upgradeSettings": {
+    "drainTimeoutInMinutes": null,
+    "maxSurge": "10%",
+    "nodeSoakDurationInMinutes": null
+  },
+  "vmSize": "standard_d2s_v3",
+  "vnetSubnetId": null,
+  "workloadRuntime": null
+}
+```
+
+As you can see the _nodeTaints_ field now updated to `null`.
+
+## Set DNS name
+
+When we created our AKS cluster, we automatically created a public IP too. Now we need to create a DNS name for the public IP. The resouce of the IP is not in our default resource group `recordingbottutorial`, because the creation of our AKS cluster created a Resource group with the managed resources. If you already know the resource group name of the managed resources, you can coninue with [getting the public IP resource name](#get-the-public-ip-resource-name).
+
+### Get the managed resources resource group name
+
+We can find the managed resource group in the description of the AKS cluster:
+
+```powershell
+az aks show
+    --resource-group recordingbottutorial
+    --name recordingbotcluster
+    --subscription "recordingbotsubscription" | Select-String 
+            -Pattern 'nodeResourceGroup' 
+            -SimpleMatch
+            -NoEmphasis
+```
+
+This will give us the response
+
+```json
+  "nodeResourceGroup": "MC_recordingbottutorial_recordingbotcluster_westeurope",
+```
+
+as the node resource group is the resource group where our cluster manges the resources, in our case the resource group is called `MC_recordingbottutorial_recordingbotcluster_westeurope`.
+
+### Get the public IP resource name
+
+Next we need the resource name of the public IP. To get the name we list the public IP resources in the managed resource group.

```

Comment:
```suggestion
Since the Public IP Address is an Azure Resource, it can be referenced by name. To find out, what name was assigned to it when it, execute the following command:

```